packages/colibri/src/documenter/documenter.ts

```diff
@@ -1,285 +1,311 @@
-// Copyright 2023
```

Comment:
why has this completely changed?

Reply:
it didn't, I must have screwed up the commit and Git can't figure out the changes. I can try and revert the commit and clean it up.

Reply:
yes, plesae